### DISCUSSION TITLE
Dropdown for entity selection

### DISCUSSION THREAD

**Discussion by Sonusss** [HASwitchPlate/openHASP#302] [Q&A]
Hi everyone,

Is there a way to manage an HA entity selector with a Dropdown object ?

I'm trying to have a page with a dropdown + slider + RGB selector where the dropdown select which RGB light to control.
I now it can be done with automation and scripts but it would be heavy code.

If possible it would be much more straightforward passing the entity ID inside the openhasp.yaml

**Comment by nagyrobi** [comment]
If you try the latest development version, you could try to accomplish something similar with the new `tag` property:
https://haswitchplate.github.io/openHASP-docs/0.6.3/integrations/home-assistant/sampl_conf/#colored-lights-panel
